.gitignore

```diff
@@ -351,3 +351,7 @@ MigrationBackup/
 
 # Ignore secrets file used for CICD build system
 local-secrets.json
+
+# Ignore Visual Studio Code files
+.vscode/launch.json
+.vscode/tasks.json
```

Comment:
Please remove these changes.  I do plan on eventually setting up some VSCode for contributors who would like to use VSCode.